src/components/TasksTable/index.jsx

```diff
@@ -365,11 +366,10 @@ export default class TasksTable extends Component {
                       component="a"
                       href={item.url}>
                       <ListItemText
-                        primary={
-                          <div className={classes.summaryText}>
-                            {item.summary}
-                          </div>
-                        }
+                        secondaryTypographyProps={{
+                          className: classes.summaryText,
+                        }}
+                        secondary={item.summary}
```

Comment:
Let's keep the summary text like before. Making it bigger doesn't make it sit gracefully with the rest of the table.